spec/message_spec.rb

```diff
@@ -3,11 +3,11 @@
 
 describe Message do
   it "should raise TypeError if passing wrong type" do
-    expect { Message.new('address', Class) }.to raise_error(TypeError)
+    -> { Message.new('address', Class) }.should raise_error(TypeError)
```

Comment:
`expect { ... }.to raise_error(...)` is preferable, otherwise Rspec monkeypatches everywhere.